### PR TITLE
Changed story content to authors intended format.

### PR DESCRIPTION
**Pull request by vanessalei411**
in this PR the story content should be formatted in its intended way by the author.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ MarkupSafe = "==0.23"
 SQLAlchemy = "==1.1.5"
 Werkzeug = "==0.11.15"
 WTForms = "==2.1"
-psycopg2 = "*"
+# psycopg2 = "*"
```
Comment (johnyu95) [inline Pipfile]: Revert `Pipfile` and `Pipfile.lock`

File: app/share/views.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def new():
                                     activist_start=form.activist_start.data,
```
Comment (johnyu95) [inline app/share/views.py]: Add `escape()` to all user inputs

File: app/static/styles/womens_activism.css [hunk 3]
```diff
@@ -2291,4 +2291,43 @@ p.interact, p.timePosted {
 
 .tag-remove{
     color: #630000;
+<<<<<<< Updated upstream
```
Comment (johnyu95) [inline app/static/styles/womens_activism.css]: Remove this.

File: app/static/styles/womens_activism.css [hunk 4]
```diff
@@ -2291,4 +2291,43 @@ p.interact, p.timePosted {
...
+=======
+}
+
+/* load-more button */
```
Comment (johnyu95) [inline app/static/styles/womens_activism.css]: Remove unused CSS styles.

File: app/static/styles/womens_activism.css [hunk 5]
```diff
@@ -2291,4 +2291,43 @@ p.interact, p.timePosted {
...
+    justify-content: center;
+}
+
+#no-more-results {
+    margin: 5rem 0 0 0;
+    text-align: center;
+    font-size: 26px;
+}
+
+.request-description-text {
```
Comment (johnyu95) [inline app/static/styles/womens_activism.css]: Can you change this to `story-content-text`

File: app/templates/stories/view.html [hunk 6]
```diff
@@ -29,27 +29,27 @@ <h1>{{ story.activist_first }} {{ story.activist_last }}</h1>
                                 <iframe class="youtube-player" type="text/html" src="{{ video_url }}" frameborder="0"
                                             allowFullScreen></iframe>
                             </div>
-                            <p>{{ story.content }}</p>
+                            <p class = "request-description-text" >{{ story.content | safe }}</p>
```
Comment (johnyu95) [inline app/templates/stories/view.html]: Update class names to `story-content-text`

**Review by johnyu95 — CHANGES_REQUESTED** [review]

File: Pipfile (hunk 1 above)
Comment (vanessalei411) [reply Pipfile:37]: done

File: app/share/views.py (hunk 2 above)
Comment (vanessalei411) [reply app/share/views.py]: done

File: app/static/styles/womens_activism.css (hunk 3 above)
Comment (vanessalei411) [reply app/static/styles/womens_activism.css]: done

File: app/static/styles/womens_activism.css (hunk 4 above)
Comment (vanessalei411) [reply app/static/styles/womens_activism.css]: done

File: app/static/styles/womens_activism.css (hunk 5 above)
Comment (vanessalei411) [reply app/static/styles/womens_activism.css]: done

File: app/templates/stories/view.html (hunk 6 above)
Comment (vanessalei411) [reply app/templates/stories/view.html]: done

**Review by johnyu95 — APPROVED** [review]
Looks good, good job!

**Pull request merged**